src/style.css

```diff
@@ -0,0 +1,132 @@
+.our-body {
```

Comment:
don't add class name to body tag, you can style it by tag name
+ don't use abstract names like `our-body`

Reply:
Thanks, done

Reply:
thanks, done